enforcer-rules/src/test/java/com/google/cloud/tools/opensource/enforcer/LinkageCheckerRuleTest.java

```diff
@@ -142,21 +145,76 @@ public void testExecute_shouldFailForBadProject() throws RepositoryException {
     }
   }
 
-  @Test
-  public void testExecute_shouldPassEmptyBom() throws EnforcerRuleException {
-    rule.setDependencySection(DependencySection.DEPENDENCY_MANAGEMENT);
+  private void setupMockDependencyManagementSection(String... coordinates) {
     org.apache.maven.artifact.DefaultArtifact bomArtifact =
         new org.apache.maven.artifact.DefaultArtifact(
             "com.google.dummy",
             "dummy-bom",
             "0.1",
             "compile",
-            "jar",
+            "pom",
             "",
             new DefaultArtifactHandler());
     when(mockProject.getArtifact()).thenReturn(bomArtifact);
 
+    DependencyManagement mockDependencyManagement = mock(DependencyManagement.class);
+    when(mockProject.getDependencyManagement()).thenReturn(mockDependencyManagement);
+
+    ImmutableList<org.apache.maven.model.Dependency> bomMembers =
+        Arrays.stream(coordinates)
+            .map(DefaultArtifact::new)
+            .map(artifact -> new Dependency(artifact, "compile"))
+            .map(LinkageCheckerRuleTest::toDependency)
+            .collect(toImmutableList());
+    when(mockDependencyManagement.getDependencies()).thenReturn(bomMembers);
+
+    when(mockMavenSession.getRepositorySession()).thenReturn(repositorySystemSession);
+  }
+
+  private static org.apache.maven.model.Dependency toDependency(Dependency resolvedDependency) {
```

Comment:
I was looking for a method to convert `org.eclipse.aether.graph.Dependency` to `org.apache.maven.model.Dependency`, but could not find one.